<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movie</class>
 <widget class="QWidget" name="movie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Return">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="film1">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time1">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="place1">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="film2">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time2">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="place2">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="film3">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time3">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="place3">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="film4">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time4">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="place4">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="film5">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="time5">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="place5">
        <property name="placeholderText">
         <string>暂无</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;电影&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;地点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
